server/test/controllers/admin/AdminProgramTranslationsControllerTest.java

```diff
@@ -302,8 +308,22 @@ public void update_outOfSyncFormData_redirectsToFreshData() throws Exception {
                 .url());
     assertThat(result.flash().get(FlashKey.ERROR).get())
         .isEqualTo("The program's associated statuses are out of date.");
-
-    assertProgramNotChanged(program);
+    // latest program
+    ProgramDefinition freshProgram =
+        programRepository
+            .lookupProgram(program.id)
+            .toCompletableFuture()
+            .join()
+            .get()
+            .getProgramDefinition();
+    assertThat(
+            applicationStatusesRepository
+                .lookupActiveStatusDefinitions(freshProgram.adminName())
+                .getStatuses())
+        .isEqualTo(
+            applicationStatusesRepository
+                .lookupActiveStatusDefinitions(program.getProgramDefinition().adminName())
+                .getStatuses());
```

Comment:
Was this change intentional? I don't know what this is testing when it's part of this test.

Reply:
We testing if the data is updated or not. When there is an error, it shouldn't be updated.